paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -91,6 +94,28 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
             )
             finish()
             return
+        } else {
+            try {
+                starterArgs.config?.validate()
```

Comment:
random but why does this validation code live here and not in PaymentSheetConfigurationValidatorKtx ?

Reply:
The client secret is not part of the PaymentSheet.Configuration.  It is a separate parameter.  I will look at validating both in a single funciton.

Reply:
I moved the validation logic into the ClientSecret class so it can be shared between `PaymentSheetActivity` and `DefaultFlowController`.